README.md

```diff
@@ -76,6 +77,7 @@ Create a `manifest.json` file includes app details.
 - `isVeWorldSupported` is **required**, set this to true if your dApp supports the VeWorld browser extension, enabling its use within the VeWorld mobile wallet. Your dApp will then be listed in the dApp section of the VeWorld mobile discovery.
 - `repo` is _optional_, and must be a fully-qualified URL.
 - `contracts` is _optional_, should be **_an array_** and **_lower cases_** if provided
+- `veBetterDaoId` is **required** for VeBetterDAO DApps
```

Comment:
It should be optional considering there are VeBetterDAO and Non VeBetterDAO apps.